src/GovUk.Education.ExploreEducationStatistics.Admin/Services/Methodologies/MethodologyService.cs

```diff
@@ -123,10 +130,13 @@ public class MethodologyService : IMethodologyService
                 .OnSuccess(_userService.CheckCanAdoptMethodologyForPublication)
                 .OnSuccess(async publication =>
                 {
-                    var methodologies = await _methodologyRepository.GetUnrelatedToPublication(publication.Id);
-                    var latestVersions = await methodologies.SelectAsync(methodology =>
-                        _methodologyVersionRepository.GetLatestVersion(methodology.Id));
-                    return (await latestVersions.SelectAsync(BuildMethodologyVersionViewModel)).ToList();
+                    var publishedMethodologies = (await _methodologyRepository
+                        .GetUnrelatedToPublication(publication.Id))
+                        .Where(m => m.LatestPublishedVersionId != null);
+                    var latestVersions = await publishedMethodologies
+                        .SelectAsync(methodology => _methodologyVersionRepository.GetLatestVersion(methodology.Id));
```

Comment:
I did consider this a little when doing the work. My initial inclination was to think `GetLatestVersion` is correct, since this preserves the behaviour we had before these changes were made.

I believe this only effects the methodology details displayed when choosing a methodology to adopt:
![image](https://github.com/dfe-analytical-services/explore-education-statistics/assets/44812484/f57ed37e-7858-48d3-b67f-9ff61fa8b7c8)

With further thought, I think it is slightly confusing if it says it's unpublished, even if it says it's an amendment:
![image](https://github.com/dfe-analytical-services/explore-education-statistics/assets/44812484/23d3ddb4-ba6a-48e5-9f60-6308fba9f5e1)

Even though the wider methodology is published, it may display a methodology version that is not published when displaying adoptable methodologies. 

So I think you're right and we should only be displaying the details of the latest published methodology version.